osu.Game/Screens/Play/PlayerSettings/AudioSettings.cs

```diff
@@ -17,6 +17,7 @@ public partial class AudioSettings : PlayerSettingsGroup
         private Bindable<ScoreInfo> referenceScore { get; } = new Bindable<ScoreInfo>();
 
         private readonly PlayerCheckbox beatmapHitsoundsToggle;
+        private readonly PlayerCheckbox autoAdjustOffsetToggle;
```

Comment:
I don't think there is any reason to keep this, when removing the setting from the popup?

Reply:
removed this in [8703f59](https://github.com/ppy/osu/pull/30586/commits/8703f598ff14b5bb32149a909c3ee957635a50f6) since the file can remain unaffected by this pr